src/qibolab/instruments/zhinst.py

```diff
@@ -927,25 +930,26 @@ def measure_relax(self, exp, qubits, relaxation_time, acquisition_type):
 
                     if i == 0:
                         # Integration weights definition or load from the chip folder
-                        weights_file = (
-                            INSTRUMENTS_DATA_FOLDER
-                            / f"{self.chip}/weights/integration_weights_optimization_qubit_{q}.npy"
-                        )
+                        weights_file = weights_file = KERNELS_FOLDER / f"{self.chip}/weights/kernels.npz"
                         if weights_file.is_file():
-                            samples = np.load(
-                                weights_file,
-                                allow_pickle=True,
-                            )
+                            from qibocal.auto.serialize import load
```

Comment:
Why are you importing `qibocal` in Qibolab?

Reply:
I don't remenber, I removed it already hahaha